src/patternfly/components/Wizard/wizard.scss

```diff
@@ -0,0 +1,419 @@
+  --pf-c-wizard__close--Right: var(--pf-global--spacer--md);
+  --pf-c-wizard__close--FontSize: var(--pf-global--FontSize--xl);
+
+  // Title
+  --pf-c-wizard__title--PaddingRight: var(--pf-global--spacer--xl);
+
+  // Description
+  --pf-c-wizard__description--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-wizard__description--Color: var(--pf-global--Color--400);
+
+  // Steps
+  --pf-c-wizard__steps--md--BoxShadow: var(--pf-global--BoxShadow--md-right);
+
+  // Toggle
+  --pf-c-wizard__toggle--PaddingTop: var(--pf-global--spacer--lg);
+  --pf-c-wizard__toggle--PaddingRight: var(--pf-global--spacer--xl);
+  --pf-c-wizard__toggle--PaddingBottom: var(--pf-global--spacer--lg);
+  --pf-c-wizard__toggle--PaddingLeft: calc(var(--pf-global--spacer--xl) + var(--pf-c-wizard__nav-link--before--Width) + var(--pf-global--spacer--sm));
+  --pf-c-wizard__toggle--BoxShadow: var(--pf-global--BoxShadow--md-bottom);
+  --pf-c-wizard__toggle--ZIndex: var(--pf-global--ZIndex--sm);
+
+  // Toggle List
+  --pf-c-wizard__toggle-list--MarginRight: var(--pf-global--spacer--sm);
+  --pf-c-wizard__toggle-list--MarginBottom: calc(var(--pf-c-wizard__toggle-list-item--MarginBottom) * -1);
+
+  // Toggle List Item
+  --pf-c-wizard__toggle-list-item--not-first-child--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-wizard__toggle-list-item--not-last-child--MarginRight: var(--pf-global--spacer--sm);
+  --pf-c-wizard__toggle-list-item--MarginBottom: var(--pf-global--spacer--xs);
+
+  // Toggle Number
+  --pf-c-wizard__toggle-num--MarginRight: var(--pf-global--spacer--sm);
+
+  // Toggle Divider
+  --pf-c-wizard__toggle-divider--MarginLeft: var(--pf-global--spacer--sm);
+
+  // Toggle Icon
+  --pf-c-wizard__toggle-icon--MarginTop: #{pf-size-prem(3px)};
+
+  // Nav
+  --pf-c-wizard__nav--Top: 100%;
+  --pf-c-wizard__nav--MaxHeight: #{pf-size-prem(200px)};
+  --pf-c-wizard__nav--BoxShadow: var(--pf-global--BoxShadow--lg-bottom);
+  --pf-c-wizard__nav--md--Top: 0;
+  --pf-c-wizard__nav--md--MaxHeight: none;
+  --pf-c-wizard__nav--md--BoxShadow: none;
+  --pf-c-wizard__nav--PaddingTop: var(--pf-global--spacer--lg);
+  --pf-c-wizard__nav--PaddingRight: var(--pf-global--spacer--xl);
+  --pf-c-wizard__nav--PaddingBottom: var(--pf-global--spacer--lg);
+  --pf-c-wizard__nav--PaddingLeft: calc(var(--pf-global--spacer--xl) + var(--pf-c-wizard__nav-link--before--Width) + var(--pf-global--spacer--sm));
+  --pf-c-wizard__nav--ZIndex: var(--pf-global--ZIndex--xs);
+
+  @media screen and (min-width: $pf-global--breakpoint--md) {
+    --pf-c-wizard__nav--Top: var(--pf-c-wizard__nav--md--Top);
+    --pf-c-wizard__nav--MaxHeight: var(--pf-c-wizard__nav--md--MaxHeight);
+    --pf-c-wizard__nav--BoxShadow: var(--pf-c-wizard__nav--md--BoxShadow);
+  }
+
+  // Nav List (nested)
+  --pf-c-wizard__nav--nested--MarginLeft: calc(var(--pf-c-wizard__nav-link--PaddingLeft) + var(--pf-global--spacer--md));
```

Comment:
Where is this variable - `--pf-c-wizard__nav-link--PaddingLeft`. Might be wrong but I can't see it

Reply:
nice, the build was failing because of the missing var :) that's a leftover from a previous change. addressed that and another var.